test/test-tarantool/proxy.lua

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env tarantool
```

Comment:
According to the [code](https://github.com/tarantool/test-run/blob/d986990b9a8a8f48f05ba03690877c4e5823aa28/lib/preprocessor.py#L329), it will not be used.

Reply:
removed.

Reply:
Hm. It is here now.